acceptance/tests/partitions/partitions_test.go

```diff
@@ -86,151 +86,134 @@ func TestPartitions(t *testing.T) {
 
 			ctx := context.Background()
 
-			serverHelmValues := map[string]string{
-				"global.datacenter": "dc1",
-
+			commonHelmValues := map[string]string{
 				"global.adminPartitions.enabled": "true",
-				"global.enableConsulNamespaces":  "true",
-				"global.tls.enabled":             "true",
-				"global.tls.httpsOnly":           strconv.FormatBool(c.secure),
-				"global.tls.enableAutoEncrypt":   strconv.FormatBool(c.secure),
 
-				"server.exposeGossipAndRPCPorts": "true",
+				"global.enableConsulNamespaces": "true",
+
+				"global.tls.enabled":           "true",
+				"global.tls.httpsOnly":         strconv.FormatBool(c.ACLsAndAutoEncryptEnabled),
+				"global.tls.enableAutoEncrypt": strconv.FormatBool(c.ACLsAndAutoEncryptEnabled),
+
+				"global.acls.manageSystemACLs": strconv.FormatBool(c.ACLsAndAutoEncryptEnabled),
 
 				"connectInject.enabled": "true",
 				// When mirroringK8S is set, this setting is ignored.
 				"connectInject.consulNamespaces.consulDestinationNamespace": c.destinationNamespace,
 				"connectInject.consulNamespaces.mirroringK8S":               strconv.FormatBool(c.mirrorK8S),
 
-				"global.acls.manageSystemACLs": strconv.FormatBool(c.secure),
-
 				"meshGateway.enabled":  "true",
 				"meshGateway.replicas": "1",
 
 				"controller.enabled": "true",
 
-				"dns.enabled": "true",
+				"dns.enabled":           "true",
+				"dns.enableRedirection": strconv.FormatBool(cfg.EnableTransparentProxy),
```

Comment:
I think the EKS tests passed without having dns redirection enabled in the server cluster-- I guess that makes sense because we're making the tproxy request from the client cluster but probably makes sense to have this enabled on all clusters in case we ever change this test to make tproxy requests from multiple places

Reply:
I think the redirection is further down in that file. Line 122-125ish.

Reply:
Oh I think we had it before in a separate if condition. I just moved it and changed to be use `FormatBool` based on config to simplify a bit

Reply:
Iryna made it better